utils/mtpoe-ctrl/Makefile

```diff
@@ -0,0 +1,44 @@
+#
+# Copyright (C) 2018 adron@yapic.net
+#
+
+include $(TOPDIR)/rules.mk
+
+PKG_NAME:=mtpoe-ctrl
+PKG_SOURCE_VERSION:=9c42b74274cca10810363dbbb6d161bfb9cedb7e
+PKG_MIRROR_HASH:=1ac2902161ba4bb1e8af397aa48821ccea95740199b1c155f98161c7ff3a93df
+PKG_VERSION:=1.15
+PKG_RELEASE:=18
+
+PKG_SOURCE_PROTO:=git
+PKG_SOURCE_URL:=https://github.com/prudy/mtpoe_ctrl.git
+PKG_SOURCE:=$(PKG_NAME)-$(PKG_VERSION).tar.gz
+PKG_SOURCE_SUBDIR:=$(PKG_NAME)-$(PKG_VERSION)
+
+PKG_MAINTAINER:=Sergey Sergeev <adron@yapic.net>
+PKG_LICENSE:=GPL-2.0
+
+include $(INCLUDE_DIR)/package.mk
+
+define Package/mtpoe-ctrl
+  SECTION:=utils
+  CATEGORY:=Utilities
+  DEPENDS:=+libuci +kmod-spi-dev
+  TITLE:=Management utility for Mikrotiks PoE V2/V3/V4
+  URL:=$(PKG_SOURCE_URL)
+endef
+
+define Package/mtpoe-ctrl/description
+	Management utility for Mikrotiks PoE V2/V3/V4
```

Comment:
This description is the same as TITLE. :-( Not very informative.

Reply:
I did not touch it, although it makes sense to change it.